design.md

```diff
@@ -1,13 +1,37 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+The target for the design is users who want to program Picobot and have either a basic knowledge of 
+programming or logic flow.
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+We converted Picobot's implicit conditionals into explicit 'if' statements which programmers will be familiar
+with and non-programmers may find closer to natural language and thus more readable. We also made the movememnts
+more explicit by forcing the user to type 'move' and a direction when specifying a move. We think this will
+better distinguish an action taken by the robot from a conditional check of the robot's surroundings. We created funtion names such as 'isEmpty(Direction)' and 'isWall(Direction)' to clarify how the robot checks its surroundings.
+Our overall design principle is valuing code clarity and readability over brevity so the language is more 
+accessible to novice programmers. We also chose to include task which set an initial state and allow the user
+to group rules together. We think this will allow users to write more modular code by grouping related behaviors.  
 
```

Comment:
I like your use of "if", though the one thing that somewhat confused me was the optional "next" keyword. I wasn't sure at first, looking at the ideal syntax example, what was going on when a rule didn't have a next rule.